docs/faq.md

```diff
@@ -77,11 +77,15 @@ good to them.
 
 ## I'd like to get my fuzzer evaluated on the free FuzzBench service, but I don't want the results to be public yet.
 
-Please reach out to us at fuzzbench@google.com.
+Please reach out to us at fuzzbench@google.com. If we agree to benchmark your
+fuzzer, please send us a patch file for fuzzbench and please test your patch
```

Comment:
maybe add "that applies cleanly on master"

Reply:
Done.